SortedArrayUnknownSize.cpp

```diff
@@ -0,0 +1,41 @@
+// Time Complexity - It is O(logn) since we are finding the target using the Binary Search and in one pass.
+// T.C. for finding the range is also log(n) since we are doing 2x at every iteration.
+// Space Complexity - It is O(n) since the search depends on the number of elements in the array?
```

Comment:
Again here Space complexity is O(1) as we are using any extra space for searching an element.